### PR TITLE
GraphQL Reporting Mutations

### PR DESCRIPTION
**Pull request by alflennik**
Implements mutations using the updated Reporting-aware GraphQL schema.

- ⚠️ Deletes database contents
- Heavily refactors the import script, so I manually copied over [the code from Howard's branch that implements the title import update](https://github.com/w3c/aria-at-app/pull/320).
- This will remove the backend blocker the frontend update task Howard is working on, as well as the Reporting frontend task.

What excites me about this update is that I believe it removes nearly all traces of the legacy modeling from the backend, and opens up the chance to do the same on the frontend.

### CODE REVIEW

**Review by howard-e — APPROVED** [review]
Thanks @alflennik! There's a few things here that we've talked about internally and I've gone through and verified that those items work as expected 👍

File: server/resources/unexpectedBehaviors.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+[
```
[the remark this answers is not in the document]
Comment (alflennik) [reply server/resources/unexpectedBehaviors.json:1]: Good idea! Now that you suggest it, I'm wondering why I didn't do it that way.

File: client/utils/checkForConflict.js [hunk 2]
```diff
@@ -6,6 +6,8 @@
  */
 
 export default function(results, userId) {
+    throw new Error('Handled by GraphQL');
```
[the remark this answers is not in the document]
Comment (alflennik) [reply client/utils/checkForConflict.js:9]: Sure!

[the remark this answers is not in the document]
Comment (alflennik) [reply client/utils/checkForConflict.js:9]: Ok, deleting these files ended up being more difficult than I expected - I also had to delete all the files referencing this function, and all the files referencing those functions, and eventually my deletions started overlapping with the refactor PR that's currently open, so I think I have to leave this one as is!

File: server/resources/unexpectedBehaviors.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (alflennik) [reply server/resources/unexpectedBehaviors.json:1]: Done!

**Pull request closed without merging**